modules/builtin/src/bot-templates/learn-botpress/bot.config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "../../bot.config.schema.json",
+  "description": "A bot with a simple workflow made for learning Botpress.",
+  "active": true,
+  "version": "12.26.3",
```

Comment:
Do we want migrations to run on this template?

Reply:
Other bot templates use version 1.0.0

Reply:
Migrations don't need to run here since the template was created on this version, and when we merge the new update mechanism the studio, it won't matter anymore